README.md

```diff
@@ -23,14 +23,73 @@ practices.
 ## Versioning
 The commit hash of the deployed code is considered its version identifier. The commit hash can be retrieved locally via `git rev-parse HEAD`.
 
-## Setting Up
+## Setting Up Your Local Dev Environment
 
 Contile uses Rust, and requires the latest stable iteration. See
 [rustup.rs](https://rustup.rs/) for how to install this application.
 
 Once Rust is installed you can compile using `cargo build`. This will
 create a development release.
 
+Additionally, Contile requires a connection to the ADM servers. These use several 
+keys (which are specified in the configuration file) to pull results.  The contract
+tests include a stub application that can provide ADM-like returns. See the 
+[adm_settings_test.json](./adm_settings_test.json) file for reference.
+
+Normally, these configurations run as a part of a `docker-compose` command, but the individual
+Docker files can be run by themselves.
+
+To start the ADM stub, you can run the following from the root of the Contile source repository:
+
+```shell
+    docker run \
+    --env PORT=5000 \
+    --env RESPONSES_DIR=/tmp/partner/ \
+    --env ACCEPTED_MOBILE_FORM_FACTORS=phone,tablet \
+    --env ACCEPTED_DESKTOP_FORM_FACTORS=desktop \
+    -v `pwd`/test-engineering/contract-tests/volumes/partner:/tmp/partner \
+    -p 5000:5000 \
+    mozilla/contile-integration-tests-partner
+```
+
+To use the above, you'll need to set the Contile configuration option as follows:
+`adm_endpoint_url`  = `http://localhost:5000/tilesp/desktop`. It may be helpful to
+create a settings file to contain all of the preferred settings, like the example
+of `sa-test.toml` below:
+
+```toml
+# sample stand-alone config file "sa-test.toml"
+
+debug=true
+port=8000
+host="localhost"
+human_logs=true
+maxmindddb_loc="mmdb/GeoLite2-City-Test.mmdb"
+
+adm_endpoint_url="http://localhost:5000/tilesp/desktop"
+adm_defaults='{"image_hosts":["example.com","example.org"],"impression_hosts":["example.com","example.org"],"click_hosts":["example.com","example.org"]}'
+adm_settings="test-engineering/contract-tests/volumes/contile/adm_settings.json"
+adm_partner_id="partner_id_test"
+adm_sub1="sub1_test"
+adm_has_legacy_image='["Example ORG","Example COM"]'
+adm_timeout=2
+tiles_ttl=0
+```
```

Comment:
I believe the contile gets cranky and doesn't start up without a sov source. You would need to add `sov_source`  probably easiest to use the the contract test example: https://github.com/mozilla-services/contile/blob/main/test-engineering/contract-tests/volumes/contile/sov_settings.json

Reply:
Great, thank you for mentioning that @tiftran , I'll make sure to update here and in the runbook as it's not specified there either.